packages/neutrino/src/neutrino.js

```diff
@@ -67,18 +47,18 @@ class Neutrino extends EventEmitter {
   }
 
   build(args) {
-    return this.runCommand('build', args, () => this.builder());
+    return this.runCommand('build', args, build);
   }
 
   start(args) {
-    return this.runCommand('start', args, () => (this.getWebpackOptions().devServer ? this.devServer() : this.watcher()));
+    return this.runCommand('start', args, this.getWebpackConfig().devServer ? develop : watch);
   }
 
   test(args) {
     return this.runCommand('test', args);
   }
 
-  runCommand(command, args = {}, fn) {
+  runCommand(command, args = {}, fn = noop) {
```

Comment:
We probably don't need a special `noop` here, we can either use `identity` from Ramda, or `Function.prototype`:

```js
runCommand(command, args = {}, fn = identity) {
```

```js
runCommand(command, args = {}, fn = Function.prototype) {
```